seed/docs/templates/docs/faq.html

```diff
@@ -0,0 +1,56 @@
+{% extends "seed/base.html" %}{% load compress %}
+
+{% block js %}{{ block.super }}
+{% endblock %}
+
+{% block content %}
+<div ng-controller="menu_controller" class="ng-cloak">
+    
+    {% include "seed/_sidebar.html" %}
+    
+    <div class="display">
+        <div ng-class="{narrow: narrow_controller, wide: wide_controller, content: true, hide_transition:is_initial_state}">
+            
+            {% include "seed/_header.html" %}
+            
+            <div class="page">
+                {{ faq_data|json_script:"faq-data" }}
```

Comment:
safe way to inject JSON into template. https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#json-script